src/components/TilePicker.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import * as db from "../firebase";
+import { useForm } from "react-hook-form";
+
+export function TilePicker(props: {
+  tileName: string,
+  tileInfo: db.TileInfo,
+  map: db.MapRepr,
+  tiles: db.TileDict,
+  selectedRow?: number,
```

Comment:
These should not be undefined. We shouldn't render the editor if nothing is selected.